src/components/Menu/ReactSelectStyles/menu-styles.ts

```diff
@@ -7,4 +7,7 @@ export const menuStyles: StylesConfigFunction<MenuProps<ReactSelectOption, false
 ) => ({
   ...provided,
   zIndex: 12,
+  width: 300,
```

Comment:
The ticket specifies 'on laptop/desktop', is it possible to change this so that it only applies on laptop/desktop?